src/components/ProgramRecord/ProgramRecordActions.jsx

```diff
@@ -268,7 +286,7 @@ ProgramRecordActions.propTypes = {
   renderBackButton: PropTypes.func.isRequired,
   username: PropTypes.string.isRequired,
   programUUID: PropTypes.string.isRequired,
-  sharedRecordUUID: PropTypes.string.isRequired,
+  sharedRecordUUID: PropTypes.string,
```

Comment:
If we make this not required, it is probably good to have it default to an empty string. We can do that with `defaultProps`. Here is an example in the [ProgramRecordAlert component](https://github.com/openedx/frontend-app-learner-record/blob/master/src/components/ProgramRecordAlert/ProgramRecordAlert.jsx#L122).

Apologies for not catching this last week!